client/src/features/home/LandingPage.js

```diff
@@ -10,16 +10,48 @@ import HamburgerNav from "./HamburgerNav";
 const LandingPage = () => {
   const cards = [
     {
-      heading: "Order1",
-      description: "Descption1",
+      heading: "Join 100devs",
+      description: (
+        <>
+          <a
+            className="underline font-bold"
+            href={`https://leonnoel.com/100devs/`}
+          >
+            100Devs
+          </a>{" "}
+          is a diverse and inclusive community of participants in a software
+          engineering training program run by Leon Noel. It is 100% free and
+          requires absolutely no previous technical experience.
+          </>
+      )
     },
     {
-      heading: "Order2",
-      description: "Descption2",
+      heading: "Contributing to Together",
+      description: "We are currently working on video resources for individuals that would like to learn more about how to contribute to Together and how Together works. Follow along on our Youtube when it is ready!",
     },
     {
-      heading: "Order3",
-      description: "Descption3",
+      heading: "Community Taught",
+      description: (
+        <>
+        Keep track of your 100devs course progress using 
+        {" "}
```

Comment:
nit: indent these two lines  one level deeper to match the rest of the content inside the `<>`/`</>` tags